### DISCUSSION TITLE
US Version of FS Libre 3 ?

### DISCUSSION THREAD

**Discussion by chrisdex** [maheini/FreeStyle-Libre-3-patch#35] [General]
Hi All

I've made the EU version of the libre 3 work for me and enjoyed it very much. However,  it did not work for the US version .. ie I couldn't find a patched US libre app. Does this solution solve this problem by starting a sensor on the regular app and then switching to jogluco/xdrip ? Or is there a patched US libre app available now ?

Thank you !!
Cheers 
Christian

**Comment by maheini** [comment]
Hi @chrisdex 

As you already noticed, Juggluco ist your choice in this case. There is no Patched Libre3 app which is working for US Libre. You just simply start the sensor with the original app, kill it and restart inside Juggluco.

**Reply by chrisdex** [reply]
Hi Maheini !
That's great news ! Finally a solution to use the FSL 3 ! 
I already have Juggluco installed from before when i was running a german FSL 3 on my phone using the patched L3 app. I assume i can i just use that installation without making much changes to it ? And then Juggluco feeds the data into Xdrip or can it feed directly into AAPS ?

Very happy about this !! ;) Thanks much !! 
Cheers,
Christian
